composer.json

```diff
@@ -10,6 +10,30 @@
     "drupal": {
       "type": "composer",
       "url": "https://packages.drupal.org/8"
+    },
+    "ckeditor-plugin/link": {
+      "type": "package",
+      "package": {
+        "name": "ckeditor-plugin/link",
+        "version": "4.19.3",
+        "type": "drupal-library",
+        "dist": {
+          "url": "https://github.com/yalesites-org/ckeditor-advanced-link-plugin/archive/refs/heads/main.zip",
```

Comment:
Great approach and I love what you did with the icon. My only concern is that this repo you are pointing to is not actually a fork. It is a copy with no relationship to the original project. So if there are updates, we can not pull them in. HOWEVER, this will get replaced with CKEditor5 in the near future, so I don't think that I am concerned.

Reply:
Yah, I thought about that but to fork it would have caused the entire CKEditor 4 repo to come in, unless you know of a way to just fork a specific directory?